platform_plugin_ontask/static/ontask.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    <iframe src="{{ONTASK_URL}}" frameborder="0" width="100%", height="720px"></iframe>
```

Comment:
can we not hardcode the width & height? maybe we can use something like this: https://github.com/eduNEXT/xblock-limesurvey/blob/main/limesurvey/static/html/instructor.html#L2